src/services/ansibleLint.ts

```diff
@@ -17,10 +17,11 @@ import { parseAllDocuments } from 'yaml';
 import { IAnsibleLintConfig } from '../interfaces/ansibleLintConfig';
 import { fileExists, hasOwnProperty, withInterpreter } from '../utils/misc';
 import { WorkspaceFolderContext } from './workspaceManager';
+
 const exec = promisify(child_process.exec);
 
 /**
- * Acts as and interface to ansible-lint and a cache of its output.
+ * Acts as an interface to ansible-lint and a cache of its output.
```

Comment:
This is also unrelated and could be merged a lot quicker than this PR gets enough approvals, if submitted separately.

Reply:
Sure. Makes sense.

Reply:
I think this is addressed now.